packages/react-core/src/components/Card/Card.tsx

```diff
@@ -90,22 +106,17 @@ export const Card: React.FunctionComponent<CardProps> = ({
         id={id}
         className={css(
           styles.card,
-          isHoverable && styles.modifiers.hoverable,
```

Comment:
@mcoker, could someone apply `pf-m-hoverable`.  Without, `isSelectable`.  I think removing support completely would technically be breaking.  I think we can deprecate it, but we still need to apply the style.

Reply:
@tlabaj yep, they're separate. While you could use them together, it isn't necessary since `isSelected` has its own hoverable styles (they're the same as what `isHoverable` applies). https://github.com/patternfly/patternfly/blob/dc6cf07d69cbb80026dca7fe43dc28704aabea64/src/patternfly/components/Card/card.scss#L26-L27